DocumentDb/Microsoft.DataTransfer.DocumentDb.FunctionalTests/packages.config

```diff
@@ -1,9 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Castle.Core" version="3.3.3" targetFramework="net452" />
-  <package id="EnterpriseLibrary.TransientFaultHandling" version="6.0.1304.0" targetFramework="net45" />
```

Comment:
are you sure line 4 was also removed in my change? 
I thought it was only the Microsoft.Azure.DocumentDB.TransientFaultHandling package (line 6)